devtools/create_project/xcode.cpp

```diff
@@ -582,6 +583,7 @@ void XcodeProvider::setupFrameworksBuildPhase(const BuildSetup &setup) {
 	frameworks_iOS.push_back("AudioToolbox.framework");
 	frameworks_iOS.push_back("QuartzCore.framework");
 	frameworks_iOS.push_back("OpenGLES.framework");
+	frameworks_iOS.push_back("Accelerate.framework");
```

Comment:
To avoid breaking the compilation when using configure/make (for example on buildbot), please also add it to:
 - The `LIBS` in `configure` for the ios7 backend case.
 - The link command for the `iphone` target in `ports.mk`